etc/openldap/local-export.ldif

```diff
@@ -450,4 +473,5 @@ member: cn=mbaluch,ou=users,dc=redhat,dc=com
 member: cn=jshepher,ou=users,dc=redhat,dc=com
 member: cn=osoukup,ou=users,dc=redhat,dc=com
 member: cn=atorresj,ou=users,dc=redhat,dc=com
-objectclass: groupOfNames
+member: cn=costaconrado,ou=users,dc=redhat,dc=com
+objectclass: groupOfNames
```

Comment:
Maybe we should make some general rule to always include the newline at the end of the file since then changes like this, eg. removing a newline adding a newline (I do have a settings in my editor to always include the new line), happens. What do you think guys? @osidb-dev

Reply:
I am for always ending with the new line